pkgs/development/misc/juce/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+# Native build inputs
+, cmake
+, pkg-config
+, makeWrapper
+
+# Dependencies
+, alsa-lib
+, freetype
+, curl
+, libglvnd
+, webkitgtk
+, pcre
+, darwin
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "juce";
+  version = "7.0.7";
+
+  src = fetchFromGitHub {
+    owner = "juce-framework";
+    repo = "juce";
+    rev = finalAttrs.version;
+    hash = "sha256-r+Wf/skPDexm3rsrVBoWrygKvV9HGlCQd7r0iHr9avM=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    makeWrapper
+  ];
+
+  buildInputs = [
+    freetype # libfreetype.so
+    curl # libcurl.so
+    stdenv.cc.cc.lib # libstdc++.so libgcc_s.so
+    pcre # libpcre2.pc
+  ] ++ lib.optionals stdenv.isLinux [
+    alsa-lib # libasound.so
+    libglvnd # libGL.so
+    webkitgtk # webkit2gtk-4.0
+  ] ++ lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk_11_0.frameworks.Cocoa
+    darwin.apple_sdk_11_0.frameworks.MetalKit
+    darwin.apple_sdk_11_0.frameworks.WebKit
+  ];
+
+  postInstall = ''
+    mv $out/bin/JUCE-${finalAttrs.version}/* $out/bin
+    rmdir $out/bin/JUCE-${finalAttrs.version}
+    rm -rf $out/include
+    rm -rf $out/lib
+  '';
+
+  meta = with lib; {
+    description = "JUCE is an open-source cross-platform C++ application framework for desktop and mobile applications, including VST, VST3, AU, AUv3, RTAS and AAX audio plug-ins";
```

Comment:
Looks like it should be placed in `longDescription`.

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/253561/commits/1b4aad26453a5812339c04bb3a3019618603814a